doc/installation.rst

```diff
@@ -47,122 +47,16 @@ Compute node image creation
 
 The creation and configuration of an OS image for the compute nodes uses the same tool and a similar configuration file than for the controller. While the controller configuration applies its setting to the machine on which it runs, the image configuration does so in a directory that will contain the whole image of the compute node.
 
-.. note:: Building a new image isn't required for most system administration tasks. One of the images existing on your system can be cloned and modified, then added to the provisioning tool. Creating a new image is only useful for an initial installation, or when desiring to start from a clean image.
```

Comment:
`Creating a new image is only useful for an initial installation, or when desiring to start from a clean image.` Or when you go full-Ansible mode, which is why you may want to have more than one image playbook